pkg/ruler/manager_test.go

```diff
@@ -40,21 +45,36 @@ func TestSyncRuleGroups(t *testing.T) {
 				User:      user,
 			},
 		},
+		user1: {
+			&rulespb.RuleGroupDesc{
+				Name:      "group1",
```

Comment:
[nit] Use a different group name to better differentiate from the one of the other user.

```suggestion
				Name:      "group2",
```